src/dotnet/HQ.Server/appsettings.json

```diff
@@ -13,10 +13,8 @@
   "DB_PASSWORD": null,
   "EmailService": "Logger",
   "Server": {
-    "HangfireInMemory": true,
-    "AutoMigrate": true,
-    "WebUrl": null
-  },
+      "WebUrl": "http://hq.localhost:4200/"
+    },
   "SMTP": {
```

Comment:
Don't make changes in the committed appsettings.json unless you are adding new settings. Instead you should be applied environment-specific configuration in `appsettings.Development.json`